react-app/cypress/integration/search_page.spec.js

```diff
@@ -0,0 +1,55 @@
+/// <reference types="cypress" />
+
+context('Movies page', () => {
```

Comment:
While Cypress is a great tool for end-to-end testing and l appreciate you giving it a go, you need to bear in mind that this kind of testing is relatively expensive and for such kind of assertions (to check whether some DOM elements are present on the page), you'd be much better off with snapshot tests, keeping Cypress for testing some more complex user interactions and app behaviors.